src/components/listingPage/listingPage.component.js

```diff
@@ -281,74 +299,259 @@ class ListingPage extends Component {
+                      <img
+                        src="/images/bed.svg"
+                        alt="bed"
+                        className="listing-info-icon"
+                      />
+                      <h2>
+                        {this.state.listingBeds} Bed
+                        {this.state.listingBeds > 1 ? "s" : ""}
+                      </h2>
+                    </div>
+                    <div className="listing-info">
+                      <img
+                        src="/images/bath.svg"
+                        alt="bath"
+                        className="listing-info-icon"
+                      />
+                      <h2>
+                        {this.state.listingBaths} Bath
+                        {this.state.listingBaths > 1 ? "s" : ""}
+                      </h2>
+                    </div>
+                  </div>
+                  <div className="spacer_xs"></div>
                   {this.props.userSession ? (
-                    <>
+                    <div className="listing-contact">
+                      <div className="listing-contact-info">
+                        <img src={defaultProfile} alt="profile" />
+                        {this.state.listingUserName && (
+                          <h2>{this.state.listingUserName.name}</h2>
+                        )}
+                      </div>
                       <a href={`mailto:${this.state.hostEmail}`}>
-                        <button className="btn green" type="button">
+                        <button
+                          className="listing-button btn green"
+                          type="button"
+                        >
                           {" "}
-                        Contact Host{" "}
+                          Contact Host{" "}
                         </button>
                       </a>{" "}
-                    </>
+                    </div>
                   ) : null}
-                  <div className="spacer_xxl"></div>
                 </div>
+                <div className="listing-location">
+                  <h4 className="listing-subtitle">Location</h4>
+                  <h5 className="listing-location-text">
+                    <LocationOnIcon className="listing-location-icon" />
+                    {this.state.listingLocation}
+                  </h5>
+                </div>
+                <h4 className="listing-subtitle">Details</h4>
+                <p className="listing-description">
+                  {this.state.listingDescription}
+                </p>{" "}
+                <br />
+                {this.state.listingAmenities.length > 0 && (
+                  <div className="listing-amenities">
+                    <h4 className="listing-subtitle">Amenities</h4>
+                    <div className="listing-amenities-container">
+                      {this.state.listingAmenities.map((amenity) => {
+                        let imagepath;
+                        switch (amenity) {
```

Comment:
Two things here:
- Looks like there's a duplicate case for "Kitchen"
- We should probably add a default case with just `null` too to resolve the warning